src/object_store/src/object/disk.rs

```diff
@@ -249,6 +265,10 @@ impl ObjectStore for DiskObjectStore {
             .map_err(|e| ObjectError::disk(format!("failed to delete {}", path), e))?;
         Ok(())
     }
+
+    async fn list(&self, _prefix: &str) -> ObjectResult<Vec<ObjectMetadata>> {
+        unimplemented!()
```

Comment:
Why is it unimplemented? I think we can implement it through `listdir`.

Reply:
Fixed.